Tools/CISupport/ews-build/steps.py

```diff
@@ -1097,7 +1097,7 @@ class AnalyzeChange(buildstep.BuildStep, AddToLogMixin):
     def _get_patch(self):
         sourcestamp = self.build.getSourceStamp(self.getProperty('codebase', ''))
         if sourcestamp and sourcestamp.changes:
-            return '\n'.join(sourcestamp.changes[0].files)
+            return b'\n'.join(sourcestamp.changes[0].files.encode('utf-8'))
```

Comment:
The opposite change, actually, since `_files()` is consumed by something expecting strings, we need to convert the patch to a string.

Reply:
(the reason we haven't likely noticed the problem in that code is because it only effects patches and we only trigger the codepath if a patch contains a `ChangeLog`, which is very unlikely 8 months after we killed them)